build/bootstrap.sh

```diff
@@ -0,0 +1,220 @@
+#! /bin/bash
+
+###
+# This script handles provisioning of the Delivery Optimization client components on supported platforms
```

Comment:
modify this to read "...handles provisiong of build environment for the Delivery Optimization..."